pkg/webhook/otel/config.go

```diff
@@ -0,0 +1,35 @@
+package otel
+
+import (
+	"sync"
+
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/metric"
+	"go.opentelemetry.io/otel/trace"
+)
+
+const (
+	OpenTelemetryServiceName = "DynatraceWebhook"
```

Comment:
Wouldn't it be better to name the service with lower case letters or am I missing something here?

Reply:
fixed [a139345](https://github.com/Dynatrace/dynatrace-operator/pull/2289/commits/a139345ed45618b919f9dd4663df34270447a785)